### PR TITLE
esp tx modules skeleton

### PR DESCRIPTION
**Pull request by olliw42**
I think there should be something, but I don't have the nerves for something more full fledged, so just something :)

### CODE REVIEW

File: docs/ELRS_TX_MODULES.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# mLRS Documentation: ELRS Tx Modules #
+
+([back to main page](../README.md))
+
+mLRS supports some few ELRS Tx modules, but only for SiK telemetry use. 
+
+> [!IMPORTANT]
+> Only for SiK telemetry use means that RC control is not supported, neither via the JR bay nor via an IN port. This also means that the mLRS Lua script cannot be used, nor can the Yaapu app.
```
Comment (jlpoltrack) [inline docs/ELRS_TX_MODULES.md:8]: Should we specify here that configuration should be done by the OLED?  It seems for the 'supported' devices that is what we recommend.

File: docs/ELRS_TX_MODULES.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# mLRS Documentation: ELRS Tx Modules #
+
+([back to main page](../README.md))
+
+mLRS supports some few ELRS Tx modules, but only for SiK telemetry use.
```
[the remark this answers is not in the document]
Comment (olliw42) [reply docs/ELRS_TX_MODULES.md]: THX. done.

File: docs/ELRS_TX_MODULES.md (hunk 1 above)
Comment (olliw42) [reply docs/ELRS_TX_MODULES.md:8]: I've added a respective sentence.

**Comment by jlpoltrack** [comment]
Make sense to merge this as-is and fill out the details later?

**Comment by olliw42** [comment]
this was somewhat the idea :)

**Pull request merged**